### DISCUSSION TITLE
Add Key User Actions API

### DISCUSSION THREAD

**Discussion by belegoth** [Dynatrace/dynatrace-configuration-as-code#760] [Q&A]
Is it possible to add this API? /applications/web/{id}/keyUserActions

**Comment by UnseenWizzard** [comment]
Hi @belegoth this is not currently supported but is already a known request (e.g. here https://github.com/dynatrace-oss/dynatrace-monitoring-as-code/discussions/213).

At this time we can not provide any ETA of when this could become available, but please keep up on future releases.
